docker/infrastructure.yml

```diff
@@ -0,0 +1,38 @@
+services:
+  localstack:
+    container_name: "localstack-main"
+    image: localstack/localstack:latest
+    ports:
+      - "127.0.0.1:4566:4566"            # LocalStack Gateway
+      - "127.0.0.1:4510-4559:4510-4559"  # external services port range
+    env_file:
```

Comment:
Is this needed?

For some reason that I don't understand it causes lambda containers to fail to start on a Mac. Removing the `env_file:` from the docker compose definition gets rid of the issue.

Reply:
As discussed, Lambda functions are configured to run with debugging support enabled  by default. As such they hang and timeout if the VS Code "Attach to Remote Node.js (fws-api)" debug configuration has not been started before a Lambda function is invoked.